src/components/styles/App.css

```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css?family=Bitter|Montserrat&display=swap');
```

Comment:
You can optimize this request by adding the required font weights. Feel free to do that on another pr and mark this conversation resolved